ports/libxml2/patches/0002-Remove-library-suffix-on-Windows.patch

```diff
@@ -0,0 +1,42 @@
+From cbc4ea23f3b4cb6e6b28cc910747027ca7a3b6db Mon Sep 17 00:00:00 2001
+From: Don <don.j.olmstead@gmail.com>
+Date: Tue, 16 Nov 2021 17:03:18 -0800
+Subject: [PATCH 2/2] Remove library suffix on Windows
+
+---
+ CMakeLists.txt | 19 -------------------
+ 1 file changed, 19 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 2051621b..b444d4eb 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -496,25 +496,6 @@ set_target_properties(
+ 	VERSION ${PROJECT_VERSION}
+ )
+ 
+-if(MSVC)
```

Comment:
Can you describe the problem(s) being fixed here? Is it that things didn't find the static library or debug version of the shared library?

The description vs title seem incomplete since this appears to be changing the dynamic build as well.

Reply:
The libxml2 library wasn't detected with a static build due to the suffix so I just removed it completely.

Reply:
Did the same happen for dynamic debug builds, because this seems to be affecting those too. If so, then I think this is good, but given the title and branch name it's hard to tell.

Reply:
It didn't like the static debug build specifically.